client/extensions/woocommerce/woocommerce-services/components/shipping-classes-field/index.js

```diff
@@ -0,0 +1,88 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { uniqBy, map } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import FormFieldset from 'components/forms/form-fieldset';
+import FormLabel from 'components/forms/form-label';
+import TokenField from 'components/token-field';
+import FormSettingExplanation from 'components/forms/form-setting-explanation';
+
+export default class ShippingClassesField extends React.Component {
+	static propTypes = {
+		id: PropTypes.string.isRequired,
+		title: PropTypes.string,
+		description: PropTypes.string,
+		value: PropTypes.array.isRequired,
+		updateValue: PropTypes.func,
+		options: PropTypes.array,
+	};
+
+	render() {
+		const { id, title, description, value, placeholder, options } = this.props;
+
+		// If there are no shipping classes, no input for them is needed.
+		if ( false === options || 0 === options.length ) {
+			return null;
+		}
+
+		return (
+			<FormFieldset>
+				<FormLabel htmlFor={ id }>{ title }</FormLabel>
+				<TokenField
+					id={ id }
+					name={ id }
+					placeholder={ placeholder }
+					value={ this.prepareValueForTokenField( value ) }
+					suggestions={ uniqBy( map( options, option => option.name ) ) }
```

Comment:
It doesn't look like this needs `uniqBy` - just `uniq`. It could also take advantage of the "property shorthand" for `map` - e.g. `map( options, 'name' )`.

In general - there's a lot of mixing Lodash and native JavaScript Array methods in this file.. can we pick one?

Reply:
Yes, `uniq` works too and I changed it.

And Lodash it is :)

Reply:
Not a very complicated reply :)

I did not have experience with Lodash (until before the trial started) and  while looking through the docs I somehow missed `uniq` and since `uniqBy` worked fine with an example like the one below, I left it that way.

```js
console.log( _.uniqBy( [ 'a', 'a', 'b' ] ) );
// [ 'a', 'b' ]
```